### DISCUSSION TITLE
Downgrading Fix

### DISCUSSION THREAD

**Discussion by funnypenguin100** [3kh0/ext-remover#614] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I've noticed that a lot of people are having trouble with downgrading- specifically the "esc+reload+power" part. It should be noted that the "power" part on a chromebook is usually the side button that glows (has the power symbol next to it), and not the top right part of the keyboard with the lock. Hopefully this helps

**Comment by py660** [comment]
Really? I have seen many school and non-school chromebooks have a power "key" on the top right corner of the keyboard.

**Reply by funnypenguin100** [reply]
Yes; however, sometimes the people _with_ a "lock key" on the top right might be confused about this shortcut- this fix is more of a troubleshoot

**Reply by py660** [reply]
huh, interesting.
